openshift/tools/go.mod

```diff
@@ -1,29 +1,28 @@
 module tools
 
-go 1.21
+go 1.22.7
 
-toolchain go1.21.5
+toolchain go1.23.0
 
-require github.com/openshift/cluster-capi-operator/manifests-gen v0.0.0-20240906154822-dbbbce4922a5
+require github.com/openshift/cluster-capi-operator/manifests-gen v0.0.0-20250603085734-2f2737e1ed64
```

Comment:
Ok I see release-4.18 is Go 1.22, so we are good here going up to 1.22.z for the build.